src/ml_tooling/transformers.py

```diff
@@ -54,23 +62,22 @@ def _most_freq(X):
                           'min': pd.DataFrame.min}
 
     def fit(self, X: pd.DataFrame, y=None):
-        if self.value is None and self.strategy is None:
-            raise TransformerError(f"Both value and strategy are set to None."
-                                   f"Please select either a value or a strategy.")
-        if self.value is not None and self.strategy is not None:
-            raise TransformerError(f"Both a value and a strategy have been selected."
-                                   f"Please select either a value or a strategy.")
         if self.strategy is not None:
```

Comment:
We can set strategy already in `__init__`

```python
def __init__(*args, **kwargs):
    self.strategy = strategy
    self.strategy_func_ = self.func_map.get(strategy) # Will be None if key is not found
    
```

Reply:
We can use truthy/falsy values

```suggestion
        if self.strategy:
```